console/src/main/java/org/web3j/console/project/InteractiveOptions.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.util.Optional;
+import java.util.Scanner;
+
+class InteractiveOptions {
+
+    static final String COMMAND_INTERACTIVE = "interactive";
+
+    private final Scanner scanner;
+    private final Writer writer;
+
+    InteractiveOptions() {
+        this(System.in, System.out);
+    }
+
+    InteractiveOptions(final InputStream inputStream, final OutputStream outputStream) {
+        this.scanner = new Scanner(inputStream);
+        this.writer = new PrintWriter(outputStream);
+    }
+
+    final String getProjectName() {
```

Comment:
the methods should have a visibility modifier in any case

Reply:
changed them to protected.